docs/electronics/Development_Boards/Arduino.md

```diff
@@ -167,4 +167,10 @@ Serial.begin(9600);
 
   
 
-The analogRead function reads the voltage at an analog pin and linearly converts it to a value between 0 and 1023. The Serial.println() function prints a variable to the serial monitor, followed by a newline (using Serial.print() to print data without the newline). The delay(1) is to limit the amount of data printed to the serial monitor.
\ No newline at end of file
+The analogRead function reads the voltage at an analog pin and linearly converts it to a value between 0 and 1023. The Serial.println() function prints a variable to the serial monitor, followed by a newline (using Serial.print() to print data without the newline). The delay(1) is to limit the amount of data printed to the serial monitor.
+
+  
+
+  
+
+For more tutorials on using Arduino IDE refer to the \[Basics of Electronics\]() section.
```

Comment:
This should be just [Basics of Electronics]() (no escape sequences). I just wanted you to create an empty link